app/models/concerns/authenticatable.rb

```diff
@@ -8,4 +8,8 @@ module Authenticatable
   def send_authentication_email
```

Comment:
no need to create concern if using in single file
Check if we can move these methods in user.rb

Reply:
Done